static/templates/bookend.hbs

```diff
@@ -9,11 +9,23 @@
             {{#if deactivated}}
                 {{t "This stream has been deactivated" }}
             {{else if subscribed }}
-                {{#tr}}You subscribed to stream {stream_name}{{/tr}}
+                {{#tr}}
+                    You subscribed to stream <z-stream-icon></z-stream-icon>
+                    {{#*inline "z-stream-icon"}}<i class="{{stream_icon}} stream-icon" aria-hidden="true"></i>{{/inline}}
+                    {{stream_name}}
+                {{/tr}}
             {{else if just_unsubscribed }}
-                {{#tr}}You unsubscribed from stream {stream_name}{{/tr}}
+                {{#tr}}
+                    You unsubscribed from stream <z-stream-icon></z-stream-icon>
+                    {{#*inline "z-stream-icon"}}<i class="{{stream_icon}} stream-icon" aria-hidden="true"></i>{{/inline}}
+                    {{stream_name}}
+                {{/tr}}
             {{else}}
-                {{#tr}}You are not subscribed to stream {stream_name}{{/tr}}
+                {{#tr}}
+                    You are not subscribed to stream <z-stream-icon></z-stream-icon>
+                    {{#*inline "z-stream-icon"}}<i class="{{stream_icon}} stream-icon" aria-hidden="true"></i>{{/inline}}
+                    {{stream_name}}
```

Comment:
Can we use `stream_privacy.hbs` here?

Reply:
@amanagr I'm not sure though I have updated the PR and description, you can re-review.

Reply:
We should use `stream_privacy.hbs` here.

Reply:
@amanagr I'm using the `stream_privacy.hbs`, and I noticed that the icon for the `#` is not displaying properly (as shown in the attached screenshot). It looks like the issue might be related to the fact that the CSS uses a span element for displaying the line, and the `#` icon in the `stream_privacy.hbs` file also uses a span element.

<img width="878" alt="Screenshot 2023-01-03 at 10 06 36 PM" src="https://user-images.githubusercontent.com/72064462/210404890-4d2b7055-e8fe-4d8f-b6f2-81ba7e5942b0.png">

Reply:
Can you push that change in a different branch and link it here so that I can help you out?

Reply:
Sure

https://github.com/akarsh-jain-790/zulip/tree/issue-23685-temp

Reply:
@akarsh-jain-790 I don't understand what are you trying to do in that branch. I wanted you to use something like this:
```html
                {{#tr}}
                    You subscribed to stream {{> stream_privacy }}
                {{/tr}}
```

Reply:
@amanagr Still getting the same result. It appears that the # icon is not displaying properly in the stream_privacy, while the globe and lock icons are functioning as expected. This discrepancy might be due to the conflicting use of span elements in both the CSS . 


> @amanagr I'm using the `stream_privacy.hbs`, and I noticed that the icon for the `#` is not displaying properly (as shown in the attached screenshot). It looks like the issue might be related to the fact that the CSS uses a span element for displaying the line, and the `#` icon in the `stream_privacy.hbs` file also uses a span element.
> 
> <img alt="Screenshot 2023-01-03 at 10 06 36 PM" width="878" src="https://user-images.githubusercontent.com/72064462/210404890-4d2b7055-e8fe-4d8f-b6f2-81ba7e5942b0.png">

<img width="892" alt="Screenshot 2023-02-07 at 12 18 02 PM" src="https://user-images.githubusercontent.com/72064462/217172435-48cda0ae-7445-4d05-862e-3986dfbab650.png">

<img width="872" alt="Screenshot 2023-02-07 at 12 38 33 PM" src="https://user-images.githubusercontent.com/72064462/217172632-20fb6f12-e492-479d-a9ce-cd2e3af0d6e3.png">

Reply:
Ok